reports/vamshi99.md

```diff
@@ -0,0 +1,209 @@
+---
+layout: report
+report_year: '2018'
+permalink: /GSoC/2018/StatusReport/VamshiKrishna
+student: Vamshi Krishna Bommerla
+organisation: coala
+organisation_link : https://coala.io
+project: Implement GitMate automations and plugins for coala
+project_link: https://summerofcode.withgoogle.com/projects/#5344316227059712
+tarball: https://rawgit.com/Vamshi99/GSoCReport/master/project.tar
+mentors: >
+ [Naveen Kumar Sangi](https://github.com/nkprince007),
+ [Lasse Schuirmann](https://github.com/sils),
+ [Manvendra Singh](https://github.com/manu-chroma),
+ [Hizkia Felix](https://github.com/HizkiFW)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/4
+ - Phase 1 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/5
+ - Phase 2 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/6
+ - Phase 3 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/7
+bio: >
+ I am third year student of B.Tech Computer Science Engineering at Amrita
+ University, Amritapuri, Kerala. I participated in GSoC and worked with
+ [coala](https://coala.io) and [GitMate](https://gitmate.io) to implement
+ GitMate plugins and automation which help coala development workflow. I worked
+ on adding plugins which auto-reject, auto-review, auto-assign PRs and notify
+ merge conflicts in a PR. I also implemented a assign request plugin to assign
+ issues to a user if he/she satisfies defined eligibilty conditions.
+social:
+ - GitHub:
+   - username: Vamshi99
+   - link: https://github.com/Vamshi99
+ - GitLab:
+   - username: Vamshi99
+   - link: https://gitlab.com/Vamshi99
+ - Gitter:
+   - username: Vamshi99
+   - link: https://gitter.im/Vamshi99
+ - OpenHub:
+   - username: Vamshi99
+   - link: https://www.openhub.net/accounts/Vamshi99
+email: bommvams72@gmail.com
+blog: https://vamshi99.blogspot.com
+activity:
+ - 0:
```

Comment:
start at `1` and https://github.com/coala/cEPs/commit/a4b2ab35243dc0b6aebc27986d33336d3bf80a64 should be the first one